poetry/core/masonry/builders/wheel.py

```diff
@@ -181,7 +181,7 @@ def _write_metadata(self, wheel):
                 relative_path = "%s/%s" % (self.dist_info, path.relative_to(self._path))
                 self._add_file(wheel, path, relative_path)
             else:
-                logger.debug("Skipping: %s", path.as_posix())
+                logger.debug("Skipping: {}".format(path.as_posix()))
```

Comment:
https://docs.python.org/3/howto/logging.html#logging-variable-data 

I am not sure why the logger printed out the `%s` literally for you. Something else going on? Either way this change should not be required.

Reply:
I couldn't figure out what's going on here. The more strange thing is, that the debug messages with "Adding: " are prepended by a `-` automatically but for "Skipping" not. As soon as I change "Adding:" to something different, the `-` disappears as well :thinking: I think we shouldn't stress that point in this PR.

Reply:
Can you raise this in another issue please? This mostlikely is some wierd config somewhere.